package.json

```diff
@@ -46,7 +49,7 @@
         "@monaco-editor/react": "^4.3.1",
         "@types/react-transition-group": "^4.4.2",
         "ajv": "^8.10.0",
-        "axios": "^0.21.1",
+        "axios": "^1.6.8",
```

Comment:
please revert this to 0.21.1

Reply:
updated the axios back to ^0.21.1 and also seeing tarball getting generated